src/data-dictionary/events/ContainerSample/memorySwapUsageBytes.md

```diff
@@ -0,0 +1,9 @@
+---
+name: memorySwapUsageBytes
+type: attribute
+units: bytes (B)
+events:
+  - ContainerSample
+---
+
+The amount of memory+swap the container is using.
```

Comment:
Same as in https://github.com/newrelic/docs-website/pull/1936/files#r619181804, personal preference.